client/locales/en-US.json

```diff
@@ -32,6 +32,13 @@
   "darkModeToggle": "Toggle dark mode",
   "dateAdded": "Date added",
   "details": "Details",
+  "dialog": {
+    "upNext": {
+      "nextItemPlayingIn": "Next Item Playing in",
```

Comment:
Avoid Title Case.

"Next item" is also kind of bad. The sentence should indicate *which* kind of item is playing. So something like "Next episode in", "Next movie in", "Next video in".  
We know it'll be playing, so I don't think "playing" is needed.